about-page.html

```diff
@@ -0,0 +1,119 @@
+</div>
+</header>
+<div id="nav-wrap">
+<div id="close-button"><img src="img/x-icon.svg" alt="close-button"></div>
+<div id="nav-m-wrap">
+  <li><a href="index.html">Home</a></li>
+  <li><a href="about-page.html">About</a></li>
+  <li><a href="#courses">Courses</a></li>
+  <li><a href="#instructor">Instrictors</a></li>
+</div>
+</div>
+
+  <main>
+    <section id="about-hero">
+      <div class="container">
+        <div class="hero">
+          <div>
+            <h1 class="fs-secondary-heading fw-bold text-accent-400 ">Break into tech with <br>the tech academy </h1>
+            <div class="about-hero-text">
+            <p>The Tech Academy is an award-winning, licensed trade school established in 2020 by Zainab Alzubair  and headquartered in Dubai, UAE.<br>
+              We deliver online tech Courses by experienced instructors.<br> We offer classes in virtually every aspect of technology, including computer programming, data science, website development, mobile app development, game development, design, cyber security, and more.
+            </p>
+            </div>
+            <p>please contact us by Email for any further questions about Tech Academy courses
+            </p>
+            <a class="mail-link"href="mailto:name@email.com">admin@admin.com</a>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section id="about-work">
+      <div class="container">
+        <div>
+          <h2 class="title title fs-700 text-neutral-900 fw-bold">Work with us</h2>
+          <div class="guide_bar"></div>
+        </div>
+        <div class="even-columns-about">
+          <div>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Corrupti vitae labore expedita?<br> Aliquam consectetur assumenda laborum ullam accusantium. Aliquid voluptatum in consequatur. Delectus maxime labore,<br> numquam voluptate fuga soluta aliquam.</p>
+          </div>
+          <div >
+            <img  class="about-logo" src="img/about_logo.png" alt="logo">
+          </div>
+        </div>
+      </div>
+    </section>
+    <hr>
+    <section>
+      <div class="container">
+        <div class="webiner">
+          <h2 class="title title fs-700 text-neutral-900 fw-bold">See the past career development webinar</h2>
+          <div class="guide_bar"></div>
+          <p>take a look at the last two tech academy webinars </p>
+        </div>
+        <div class="even-columns-webinar">
+          <div class="w-card">
+            <img  class="card-img" src="img/webiner.jpg" alt="webiner">
+             <p class="w-text fw-bold">Tech academy career webinar 2021</p>
+         
+          </div>
+          <div class="w-card">
+            <img class="card-img"src="img/webiner.jpg" alt="webiner">
+              <p class="w-text fw-bold">Tech academy career webinar 2022</p>
+            
+          </div>
+        </div>
+      </div>
+    </section>
+  </main>
+
+  <footer class="about-footer" >
+    <div class="container">
```

Comment:
![partners](https://user-images.githubusercontent.com/93039359/196927668-581166a6-3be7-4839-af18-9c5d3c1b7b2e.png)

- [ ] I guess you forgot to add the partners section in your about page in your mobile desing. Kindly add it to make your project design consistent to the template :+1: